draft/spec/index.html

```diff
@@ -349,11 +349,12 @@ <h2>Object Structure</h2>
     <section id="object-conformance-declaration">
         <h2>Object Conformance Declaration</h2>
         <p>
-            The version declaration MUST be formatted according to the [[!NAMASTE]] specification. It MUST be an empty
-            file in the base directory of the object giving the OCFL Object version in the filename. The filename MUST
-            be constructed with a leading zero-equals (<code>0=</code>) string, the string <code>ocfl_object_</code>,
-            followed by the OCFL specification version number. For example <code>0=ocfl_object_1.0</code> for version
-            1.0 of this specification.
+            The version declaration MUST be formatted according to the [[!NAMASTE]] specification. It MUST be an
+            file in the base directory of the <a>OCFL Storage Root</a> giving the OCFL version in the filename.
```

Comment:
Given the current section title, "Object Conformance Declaration", I would suggest not changing "base directory of the object" to "base directory of the OCFL Storage Root".
...unless we are no longer specifying the inclusion of a NAMASTE file in the object base.

Reply:
👍

Reply:
I think this is just a simple error, should be "OCFL Object" not "OCFL Storage Root"